www/download.php

```diff
@@ -241,7 +241,7 @@ function downloadSingleFile($transfer, $recipient, $file_id, $recently_downloade
         if (!$chunk_size)
             $chunk_size = 1024 * 1024;
 
-        if($file->encrypted_size){
+        if($file->transfer->options->encryption){
```

Comment:
This change by itself is good, though. Checking that a flag is explicitly true is easier to read than checking if a variable is not falsey.